libretroshare/src/pqi/sslfns.cc

```diff
@@ -711,67 +705,39 @@ uint32_t getX509RetroshareCertificateVersion(X509 *cert)
 	}
 }
 
-// Not dependent on sslroot. load, and detroys the X509 memory.
-int	LoadCheckX509(const char *cert_file, RsPgpId& issuerName, std::string &location, RsPeerId &userId)
+int LoadCheckX509(
+        const char* cert_file, RsPgpId& issuer, std::string& location,
+        RsPeerId& userId )
 {
-	/* This function loads the X509 certificate from the file, 
-	 * and checks the certificate 
-	 */
+	constexpr int failure = 0;
+	constexpr int success = 1;
 
 	FILE *tmpfp = RsDirUtil::rs_fopen(cert_file, "r");
-	if (tmpfp == NULL)
+	if (tmpfp == nullptr)
 	{
-#ifdef AUTHSSL_DEBUG
-		std::cerr << "sslroot::LoadCheckAndGetX509Name()";
-		std::cerr << " Failed to open Certificate File:" << cert_file;
-		std::cerr << std::endl;
-#endif
-		return 0;
+		RsErr() << __PRETTY_FUNCTION__ << " Failed to open Certificate File: "
+		        << cert_file << std::endl;
+		return failure;
 	}
 
 	// get xPGP certificate.
-	X509 *x509 = PEM_read_X509(tmpfp, NULL, NULL, NULL);
+	X509* x509 = PEM_read_X509(tmpfp, nullptr, nullptr, nullptr);
 	fclose(tmpfp);
 
-	// check the certificate.
-	bool valid = false;
-	if (x509)
+	if(!x509)
 	{
-                valid = CheckX509Certificate(x509);
```

Comment:
`CheckX509Certificate` did nothing only returned true, soI removed that from the code, because it was just misleading making people believe we where checking something while we were checking nothing as the function implementation was literally `return true;`

Reply:
That's what i'm saying! Instead of removing it, we should make it right. Now, not only is is not called but the code saying "please fix this" is gone. That's exactly what we dont want ;-)

Reply:
doesn't `AuthX509WithGPG` do that?

Reply:
resolved in 16d606b51388ddf7fa2b70dbffd31d894f52978d